src/testing/e2e/integrationtests/oneagent_controller_integration_test.go

```diff
@@ -37,10 +36,10 @@ func TestReconcileOneAgent_ReconcileOnEmptyEnvironment(t *testing.T) {
 	// Check if deamonset has been created and has correct namespace and name.
 	dsActual := &appsv1.DaemonSet{}
 
-	err = e.Client.Get(context.TODO(), types.NamespacedName{Name: oaName + "-" + daemonset.PodNameOSAgent, Namespace: DefaultTestNamespace}, dsActual)
+	err = e.Client.Get(context.TODO(), types.NamespacedName{Name: oaName + "-" + dynatracev1beta1.PodNameOSAgent, Namespace: DefaultTestNamespace}, dsActual)
```

Comment:
You could use `dynakube.OneAgentDaemonsetName()` if you change the inline creation of the dynakube in line 25 and put it into an actual variable